gallery/requirements.txt

```diff
@@ -0,0 +1,2 @@
+streamlit==1.12.2
+streamlit-extras
```

Comment:
```suggestion
streamlit-extras
streamlit-keyup
```

Reply:
Oh my bad, `streamlit-keyup` is a dependency of `streamlit-extras`

Reply:
Yeah, that's the nice thing about doing it this way. I suppose in fact we could just have streamlit-extras if it wasn't for the need to pin streamlit for the monkey-patching.